src/cfn/template.yaml

```diff
@@ -77,105 +79,15 @@ Conditions:
 
 Resources:
 
+  # Network configuration
+
   VPC:
     Type: AWS::EC2::VPC
     Properties:
       EnableDnsSupport: true
       EnableDnsHostnames: true
       CidrBlock: !Ref VpcIpBlock
 
-  ApiGatewayInvokeRole:
```

Comment:
I reorganized the things here to make more sense, but the only thing that changed are really the Containers part